src/components/static/Link.jsx

```diff
@@ -0,0 +1,21 @@
+import Link from "next/link";
+const CUstomLink = ({ text, link }) => {
+  return (
+    <Link
+      target="_blank"
+      href={link}
+      className={`no-underline hover:-translate-y-0.5 duration-300 text-white relative bg-gradient-to-r from-bear-purple to-bear-teal min-w-1/2 rounded-full`}
```

Comment:
The border on the gradient button has weird behavior (border flickers a bit between resting state and moving state?)
`scale-[1.01]` seems to fix this though (something about the transform rendering seems to make it be drawn differently?)